src/celengine/timelinephase.h

```diff
@@ -78,48 +78,41 @@ class TimelinePhase
         return m_startTime <= t && t < m_endTime;
     }
 
-    static TimelinePhase* CreateTimelinePhase(Universe& universe,
+    static TimelinePhase::SharedConstPtr CreateTimelinePhase(Universe& universe,
```

Comment:
indentation

Reply:
Again, what do you mean?

Reply:
all arguments if placed on several lines should start on the same column